examples/migrations/n8n_executions/live-roundtrip-okf/sessions/n8n-operations-proof_2026-07-30_unknown_summary.md

```
@@ -0,0 +1,109 @@
+# Session Summary for n8n-operations-proof
+**Session ID:** `unknown`
+
+---
+
+### [2026-07-30 09:00:59] [DECISION] Lead Atlas Fleet: hot (100)
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)